app/src/main/res/values/strings.xml

```diff
@@ -264,10 +286,21 @@
     <!-- Menu text for setting themes -->
     <string name="label_menu_themes">Themes</string>
    
+   
+    <!-- Theme settings -->
+    <!-- Titles in theme settings panel -->
+    <string name="theme_settings_title">Choose your theme</string>
+    <string name="theme_settings_subtitle">Save some battery by selecting a dark theme.</string>
+   
+    <!-- Dialog for asking if the user want to set as default browser-->
+    <string name="set_as_default_dialog_title">Well done!</string>
+    <string name="set_as_default_dialog_subtitle">Your Firefox Lite is looking great! Do you want to set it as your default browser?</string>
+   
+   
     <!-- Add top sites -->
     <!-- top sites categories on the top sites list -->
     <string name="top_site_category_1">Popular Websites</string>
-    <string name="top_site_category_2">Social Media</string>
```

Comment:
There is a `News` category in the top sites list.

Reply:
Can we leverage the existing "News"?

Reply:
Suggest not to reuse `label_menu_news` because they have total different context. Besides, it would be more flexible if the category name might be changed in the future. (ex `Social Media & News`)

Reply:
OK!

Reply:
If you still want to reuse the previous string, please define a new string id and reference the existing string like following:

<string name="top_site_category_2" translatable="false">@string/label_menu_news</string>

Reply:
It's fine. Let's just make them independent :)
Just submitted a new change!